docs/firefighting/index.md

```diff
@@ -827,14 +827,29 @@ Currently on ICDS (maybe on prod/india) shard allocation is disabled. In case a
   - Shard assignment can be checked via Elasticsearch [shards API](https://www.elastic.co/guide/en/elasticsearch/reference/current/cat-shards.html) or the shards graph on Elasticsearch datadog dashboard
 - If any primaries are not allocated. Use rereoute API ([official docs](https://www.elastic.co/guide/en/elasticsearch/reference/2.4/cluster-reroute.html))
   - Reroute according to existing shard allocation
-  - Example reroute command to allocate primary shard
+  - Example reroute command to allocate primary shard <br>
+    :warning: The <b>allow_primary</b> parameter will force a new empty primary shard to be allocated without any data. If a node which has a copy of the original shard (including data) rejoins the cluster later on, that data will be deleted: the old shard copy will be replaced by the new live shard copy.
```

Comment:
I think because of this, we basically never want to use allow_primary = true. Can we keep the warning not to use allow_primary = true, but remove the examples? In a firefighting situation people are too likely to copy examples without reading the warning, so rather than saying "here's how you do this thing that will delete data, but don't do it", we keep the warning but don't show them how to do the thing we're telling them not to do

Reply:
Agreed Danny. Let's remove examples and add a link to official docs which helps us to get examples in case we need them.